src/depedenciesLibrary/voice.ts

```diff
@@ -56,6 +56,10 @@ export const playAudioFile = async (fileName: string): Promise<void> => {
       audioPlayerState.isPlaying = false;
       resolve();
     });
+    audioPlayerState.process.on('error', (err) => {
+      audioPlayerState.isPlaying = false;
+      reject(err);
+    });
```

Comment:
oops

Reply:
This works i like

Reply:
no changed needed here right? Am still not familiar with node best practices